content.js

```diff
@@ -38,31 +37,64 @@ function loop(len, idp) {
     }
     return list;
 }
+function loop(i,max, idp, list) {
+    for (let i = 0; i < max-1; i++) {
+        if (document.querySelector("#pagelet_group_mall").querySelector("div").querySelectorAll("._4-u2, .mbm, ._4mrt, ._5jmm, ._5pat, ._5v3q, ._7cqq, ._4-u8")[0].getAttribute('id') === idp) {
+            return list;
+        }
+        var id = document.querySelector("#pagelet_group_mall").querySelector("div").querySelectorAll("._4-u2, .mbm, ._4mrt, ._5jmm, ._5pat, ._5v3q, ._7cqq, ._4-u8")[0].getAttribute('id');
+        var author = document.querySelectorAll('.userContentWrapper')[0].querySelector(".fwn .fcg").textContent;
+        var content = document.querySelectorAll('.userContentWrapper')[0].querySelector(".userContent").textContent;
+        if (document.querySelectorAll('.userContentWrapper')[0].querySelector(".userContent").querySelector(".text_exposed_show") !== null) {
+            content = content.replace('...', '');
+            content = content.replace('Zobacz więcej', '');
+        }
+        var date = document.querySelectorAll('.userContentWrapper')[0].querySelector('abbr._5ptz').getAttribute('title');
+        var image = [];
+        if (document.querySelectorAll('.userContentWrapper')[0].querySelectorAll('._4-eo, ._2t9n').length === 1) {
+            image = [document.querySelectorAll('.userContentWrapper')[0].querySelector('._4-eo, ._2t9n').querySelector('img').getAttribute('src')];
+        } else if (document.querySelectorAll('.userContentWrapper')[0].querySelectorAll('._5dec').length > 1) {
+            let lenOfImages = document.querySelectorAll('.userContentWrapper')[0].querySelectorAll('._5dec, ._xcx, ._487t').length;
+            for (let j = 0; j < lenOfImages; j++) {
+                image.push(document.querySelectorAll('.userContentWrapper')[0].querySelectorAll('._5dec, ._xcx, ._487t')[j].querySelector('img').getAttribute('src'));
+            }
+        }
+        let post = new Post(id, author, date, content, image);
 
+        list.push((post));
```

Comment:
Should be delete multiple brackets and I think that should write inline. `list.push(new Post(...))`